tests/Database/DatabaseQueryBuilderTest.php

```diff
@@ -858,6 +858,34 @@ public function testArrayWhereColumn()
         $this->assertEquals([], $builder->getBindings());
     }
 
+    public function testWhereFulltext()
```

Comment:
Instead of having all of the various test cases in a singe test method, I think it'd be better if we split them to shoreter methods as this method tends to grow later in the future.

Reply:
This is fine.